lib/providers/analytics.dart

```diff
@@ -52,6 +54,12 @@ class BccmAnalyticsMetaEnricher extends AnalyticsMetaEnricher {
       extraProperties['meta']['settings'] = routeData.meta[RouteMetaConstants.settingsName];
     }
 
+    final onboardingRouteArgs = route.settings.arguments.asOrNull<OnboardingScreenRouteArgs>();
```

Comment:
Is this the correct way to check if we are on the onboarding screen?

Reply:
i wouldve checked path but this is better if it works